spirv_cross.cpp

```diff
@@ -1706,6 +1706,13 @@ void Compiler::parse(const Instruction &instruction)
 
 		auto &var = set<SPIRVariable>(id, type, storage, initializer);
 
+		auto &ttype = get<SPIRType>(type);
```

Comment:
I would like a comment here stating why this is needed (HLSL does not set stuff).